python/tvm/relay/frontend/onnx.py

```diff
@@ -0,0 +1,1078 @@
+                                     _op.nn.relu(_expr.const(1.) - _op.exp(inputs[0])) +
+                                     _op.nn.relu(inputs[0]))
+
+
+class ScaledTanh(OnnxOpConverter):
+    """ Operator converter for ScaledTanh.
+    """
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        alpha = float(attr.get('alpha', 1.0))
+        beta = float(attr.get('beta', 1.0))
+        return _op.tanh(_expr.const(beta) * inputs[0]) * _expr.const(alpha)
+
+
+class SoftPlus(OnnxOpConverter):
+    """ Operator converter for SoftPlus.
+    """
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        return _op.log(_op.exp(inputs[0]) + _expr.const(1.))
+
+
+class Softsign(OnnxOpConverter):
+    """ Operator converter for Softsign.
+    """
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        return inputs[0] / (_expr.const(1.) + Absolute.get_converter(1)(inputs, attr, params))
+
+
+class Sub(Elemwise):
+    name = 'subtract'
+
+
+class Sum(OnnxOpConverter):
+    """ Operator converter for Sum.
+    """
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        # Onnx Sum Operator
+        for in_index in range(len(inputs) - 1):
+            inputs[in_index + 1] = _op.add(inputs[in_index], inputs[in_index + 1])
+
+        return inputs[len(inputs) - 1]
+
+
+class ThresholdedRelu(OnnxOpConverter):
+    """ Operator converter for ThresholdedRelu.
+    """
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        alpha = float(attr.get('alpha', 0.0))
+        alpha_tensor = _op.full_like(inputs[0], fill_value=_expr.const(alpha))
+        mask = _op.greater(inputs[0], alpha_tensor).astype("float32")
+        return inputs[0] * mask
+
+
+def _broadcast_constraint():
+
+    def _broadcast_check(attrs):
+        if attrs.get('axis', None):
+            return False
+        return True
+
+    return _broadcast_check, "Specifying broadcast axis not allowed."
+
+
+def _fully_connected(opset):
+
+    def _impl(inputs, attr, params):
+        # get number of channels
+        channels = infer_channels(inputs[1], params)
+        attr['units'] = channels
+        return AttrCvt('dense', ignores=['axis', 'axis_w'])(inputs, attr)
+
+    return _impl
+
+
+class Upsample(OnnxOpConverter):
+    """ Operator converter for Upsample (nearest mode).
+    """
+
+    @classmethod
+    def _impl_v7(cls, inputs, attr, params):
+        scales = attr.get('scales')
+        assert len(scales) == 4 and scales[0] == 1.0 and scales[1] == 1.0 and scales[2] == scales[3]
+        mode = attr.get('mode')
+        if mode == b'nearest':
+            method = "NEAREST_NEIGHBOR"
+        elif mode == b'linear':
+            method = "BILINEAR"
+        else:
+            raise ValueError("Invalid ONNX upsample mode: {}".format(mode))
+        attr = {'scale':int(scales[-1]), 'method':method, 'layout':'NCHW'}
+        return AttrCvt('upsampling')(inputs, attr)
+
+
+class Shape(OnnxOpConverter):
```

Comment:
The shape op is a workaround in NNVM. We may need to revisit in relay.
Can be disabled here and handled outside this PR.

Reply:
will disable for now